DevAcademy/Scenes/MapView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MapView.swift
+//  DevAcademy
+//
+//  Created by Pavla Beránková on 02.08.2023.
+
+//  Brno - 49.19522264287748, 16.605414965101804
+
+import SwiftUI
+import MapKit
+
+struct MapView: View {
+    @State private var region: MKCoordinateRegion = {
+       var mapCoordinates = CLLocationCoordinate2D(latitude: 49.19522264287748, longitude: 16.605414965101804)
+        var mapZoomLevel = MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)
+        var mapRegion = MKCoordinateRegion(center: mapCoordinates, span: mapZoomLevel)
+        return mapRegion
+    }()
```

Comment:
Čau Pájo, tohle bych asi napsal spíš jak je uvedeno níž. Přijde mi to jednodušší bez složených závorek, tak mrkni, jestli se ti to bude líbit:

```suggestion
    @State private var region = MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: 49.19522264287748, longitude: 16.605414965101804), span: MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1))
```

Reply:
Určitě to je jednodušší a kratší zápis. Díky.